test/e2e/specs/editor/blocks/cover.spec.js

```diff
@@ -39,18 +38,14 @@ test.describe( 'Cover', () => {
 		} );
 		await expect( blackColorSwatch ).toBeVisible();
 
-		// Get the RGB value of Black.
-		const [ blackRGB ] =
-			await coverBlockUtils.getBackgroundColorAndOpacity( coverBlock );
-
 		// Create the block by clicking selected color button.
 		await blackColorSwatch.click();
 
-		// Get the RGB value of the background dim.
-		const [ actualRGB ] =
-			await coverBlockUtils.getBackgroundColorAndOpacity( coverBlock );
-
-		expect( blackRGB ).toEqual( actualRGB );
+		// Assert that after clicking black, the background color is black.
+		await expect( coverBlock ).toHaveCSS(
+			'background-color',
+			'rgb(0, 0, 0)'
+		);
 	} );
```

Comment:
When I was refactoring this test to use `toHaveCSS` like the other ones, I saw that it wasn't really testing what it said in the description, so I just fixed it in the process.